src/css/components/files/_c-cart-card-hr.scss

```diff
@@ -0,0 +1,6 @@
+.c-cart-card-hr{
+  @include horizontal_align();
+  display: block;
+  width: 100%;
+  border-color: $medium_green;
```

Comment:
Only declaring border-color makes it so ambiguous. Where are other border properties being inherited from?